src/components/spoiler/spoiler.scss

```
@@ -0,0 +1,109 @@
+.spoiler {
+  $root: &;
+
+  & {
+    border-top: 1px solid #E6E6E6;
+  }
```

В данном случае можно отказаться от наследования селектора без потери функционала
```suggestion
.spoiler {
  $root: &;
  
  border-top: 1px solid #E6E6E6;
```